docs/UserGuide.md

```diff
@@ -93,12 +93,13 @@ Format: `list`
 
 Edits an existing person in the address book.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG] [r/REMARK]…​`
+Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG] [dt/TAG] [r/REMARK]…​`
 
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
+* When editing tags, the tags specified using `t/` will be added to the contact (cumulatively).
+* Tags can also be removed using the delete tag `dt/` prefix, followed by the tag name.
```

Comment:
Do you think it will be value-add to the User Guide by showing how delete tag works in the example?

Reply:
The syntax is similar to adding of tags, and it has already been shown in the help window so I think it may not be necessary.

Reply:
Alright